diskmaker/controllers/lv/reconcile_test.go

```diff
@@ -318,34 +347,51 @@ func getDeiveIDs() []string {
 }
 
 func createTmpDir(t *testing.T, dir, prefix string) string {
-	tmpDir, err := ioutil.TempDir(dir, prefix)
+	tmpDir, err := os.MkdirTemp(dir, prefix)
 	if err != nil {
 		t.Fatalf("error creating temp directory : %v", err)
 	}
 	return tmpDir
 }
 
-func createTmpFile(t *testing.T, dir, pattern string) *os.File {
-	tmpFile, err := ioutil.TempFile(dir, pattern)
+func createTmpFile(t *testing.T, dir, pattern string, size int64) *os.File {
+	tmpFile, err := os.CreateTemp(dir, pattern)
 	if err != nil {
 		t.Fatalf("error creating tmp file: %v", err)
 	}
+	err = tmpFile.Truncate(size)
+	if err != nil {
+		t.Fatalf("error truncating tmp file: %v", err)
+	}
 	return tmpFile
 }
 
 func hasFile(t *testing.T, dir, file string) bool {
-	files, err := ioutil.ReadDir(dir)
+	dentries, err := os.ReadDir(dir)
 	if err != nil {
 		t.Fatalf("error reading directory %s : %v", dir, err)
 	}
-	for _, f := range files {
-		if strings.Contains(f.Name(), file) {
+	for _, d := range dentries {
+		if strings.Contains(d.Name(), file) {
 			return true
 		}
 	}
 	return false
 }
 
+func formatAsExt4(t *testing.T, fname string) {
+	cmd := exec.Command("mkfs.ext4", fname)
+	err := cmd.Run()
+	if err != nil {
+		t.Fatalf("error formatting file: %v", fname)
+	}
+}
+
+func hasExt4(fname string) bool {
```

Comment:
Same here, see comment for `formatAsExt4`.

Reply:
@RomanBednar , are you OK with this simple implementation (`return cmd.Run() == nil`) or it's worthy to check ExitCode and print explicit error message if we fail for another reason, e.g. `fsck.ext4` is not in PATH? (I intentionally preferred code simplicity there)

Reply:
@mpatlasov It's the same case as with `formatAsExt4` - let's say the `fsck.ext4` command gets called with wrong path by mistake (for `-n` arg), it will just return false and will be difficult to spot and debug. Not a hard opinion on that one, but generally it's better not to hide the errors.

Reply:
OK, agreed, let's not hide the errors: https://github.com/openshift/local-storage-operator/pull/431/commits/ff29c851192776f5c89c0f807f2f8a55cfebb810#diff-043dd3dc2a6dd72e8b4213a58c4ca5b3a9361dbc44f8313756306dced58caf8dR36-R38